subsys/bluetooth/mesh/Kconfig.models

```diff
@@ -12,6 +12,31 @@ config BT_MESH_NRF_MODELS
 	  Common mesh model support modules, required by all Nordic BT mesh
 	  models.
 
+menuconfig BT_MESH_ACKNOWLEDGED_MESSAGE
+	bool "Configuration of the waiting time for acknowledged messages"
+	default y
+	select BT_MESH_NRF_MODELS
+	help
+	  The Bluetooth Mesh Profile recommends setting this timeout to
+	  a minimum of 30 seconds. However, the exact value is application specific.
+	  The final timeout is the sum of the per hop timeout multiplied
+	  on the number of hops and the base timeout
+
+config BT_MESH_MOD_ACKD_TIMEOUT_BASE
+	int "The base timeout for acknowledged messages"
+	default 3000
+	depends on BT_MESH_ACKNOWLEDGED_MESSAGE
+	help
+	  The base timeout in milliseconds that the model waits for the response
```

Comment:
Is it better to replace "response" with "acknowledgment,"?

Reply:
This is the tricky part of the specification. The message is an acknowledged message but the model waits for a response. 

> An acknowledged message is transmitted and acknowledged by each receiving element by responding to that message. The response is typically a status message.

Furthermore, there is the transport layer control message `Acknowledgement` that is used in the segmentation process. I'd prefer to use `response` to minimize misunderstanding.

IMHO Using `acknowledged` term for models is a very bad solution at all.